_pkgdown.yml

```diff
@@ -1,9 +1,72 @@
 template:
   params:
-    bootswatch: paper
+    package: tidytemplate
```

Comment:
I don't think we want to use the tidyverse branding (unless I'm missing something).

Reply:
Oh I just thought their website template looks better.

Reply:
Their website says "Please don’t use it for your own package." What do you like about it? We might be able to copy/borrow aspects.

Reply:
Update, it is now 
```
template:
  params:
    bootswatch: flatly
    
```

See commit "Not use tidytemplate".